resources/windows/wix/customactions.wxs.erb

```diff
@@ -258,6 +258,13 @@ End If
     <CustomAction Id="RemoveLegacyNssmRegistryKey" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="immediate" Return="check"/>
     <%-end-%>
 
+    <Property Id="WixSilentExec64CmdLine"  Value="&quot;C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe&quot; New-ItemProperty -Path &quot;HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem&quot;  -Name &quot;LongPathsEnabled&quot; -Value 1 -PropertyType DWORD -Force"/>
```

Comment:
Should `C:Windows` be replaced with `[%WINDIR]` as in the above action?
Also de we need to take into account x86 vs x64? I see we do this for other actions